index.js

```diff
@@ -609,6 +611,9 @@ var torrentStream = function(link, opts, cb) {
 		});
 	}
 
+	if (pauseSwarm) swarm.pause();
+	delete pauseSwarm;
```

Comment:
This will not work as you expect. You can't delete variables - only properties and array members.